docs/DESKTOP_ENVIRONMENTS.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Desktop Environment integration
+category: Concepts
+layout: default
+---
+
+# Desktop Environments
+
+NOTE: This document is work-in-progress.
+
+## Single Graphical Session
+
+To be able to manage a graphical session using systemd, there is a strong
+requirement that each user only has one such session running at a time. While
+this might not have always been the case historically, having multiple sessions
+for one user running at the same time is problematic for a number of reasons
+(including e.g. the DBus session bus being shared).
+
+Note that in principle it is possible to run a single graphical session
+across multiple logind seats in principle. i.e. when the user logs in to a
+second seat, one could assign the seats resources to the existing session,
+effectively presenting it as a single seat.
```

Comment:
> In principle it is possible to run a single graphical session across multiple logind seats, 
> and this could be an way to use more than one display per user.
> When a user logs in to a second seat, the seat resources could be assigned to the existing session,
> allowing the graphical environment to present it is a single seat. 
> Currently nothing like this is supported or even planned.

?

Reply:
if it's not supported nor planned, I'm not sure it's a good idea to mention it in the documentation.

Reply:
I do think it makes sense to mention. Not because I expect that anyone will implement it, but because the concept strengthens the expectation that there is only a single running session per user.